### PR TITLE
Use fixtures in Tests - test_twobody and test_angles

### PR DESCRIPTION
**Pull request by shreyasbapat**
Let's do this step by step. I will make several different pull requests which will eventually enable usage of fixtures in every place where it is better for us to use.

### CODE REVIEW

**Comment by Juanlu001** [comment]
If you want some early suggestions:

* Tell me about your plans, so we can discuss :)
* Start small. In the original fixtures issue it's said that "There are a lot of repeated code in the tests, mainly sample data for orbits" https://github.com/poliastro/poliastro/issues/447, so perhaps in `test_angles`, `test_twobody` or `test_propagation` there are also opportunities.
* Leave the plotting module for a separate PR, as I have some unfinished work that might affect all of it...

**Comment by shreyasbapat** [comment]
Sure. I will work on the other tests. I was just trying out. I will discuss with you today. :)

**Comment by shreyasbapat** [comment]
@Juanlu001 Please review whenever you get a chance. If you feel something is less than what it should be, please let me know, I will fix it. :)

File: src/poliastro/tests/test_twobody.py [hunk 1]
```diff
@@ -1,40 +1,32 @@
...
 
 from poliastro.bodies import Earth
-
 from poliastro.core.elements import coe2rv, coe2mee, rv2coe
 from poliastro.twobody.equinoctial import mee2coe
 
 
-def test_convert_between_coe_and_rv_is_transitive():
-    k = Earth.k.to(u.km**3 / u.s**2).value  # u.km**3 / u.s**2
+@pytest.fixture(scope="module")
```
Comment (Juanlu001) [inline src/poliastro/tests/test_twobody.py]: Didn't know about `scope="module"`, nice!

https://docs.pytest.org/en/3.8.0/fixture.html#scope-sharing-a-fixture-instance-across-tests-in-a-class-module-or-session

However, I believe that they are used for expensive fixtures, so I'd rather remove it.

File: src/poliastro/tests/tests_twobody/test_angles.py [hunk 2]
```diff
@@ -7,6 +7,24 @@
 from poliastro.twobody import angles
 
 
+@pytest.fixture(scope="module")
```
Comment (Juanlu001) [inline src/poliastro/tests/tests_twobody/test_angles.py]: In this particular case, let's use `ANGLES_DATA` just as a module variable, without using a fixture.

File: src/poliastro/tests/test_twobody.py (hunk 1 above)
Comment (shreyasbapat) [reply src/poliastro/tests/test_twobody.py]: Sure! will do so!

File: src/poliastro/tests/tests_twobody/test_angles.py (hunk 2 above)
Comment (shreyasbapat) [reply src/poliastro/tests/tests_twobody/test_angles.py]: Okay!

**Comment by shreyasbapat** [comment]
@Juanlu001  I have rebased as well as made the suggested changes :)

**Comment by Juanlu001** [comment]
Merged! Thanks for your patience :)

**Pull request merged**